docs/provider-config/aws-bedrock-with-apikey-authentication.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: "AWS Bedrock"
+description: "Learn how to set up AWS Bedrock with Cline using credentials authentication. This guide covers AWS environment setup, regional access verification, and secure integration with the Cline VS Code extension."
+---
+
+### Overview
```

Comment:
@massi-ang I did not verify this documentation but I will take your word for it.